src/sorcha/ephemeris/simulation_parsing.py

```diff
@@ -56,18 +79,19 @@ def parse_orbit_row(row, epoch, ephem, sun_dict, gm_sun):
     equatorial_coords = np.array(ecliptic_to_equatorial([ecx, ecy, ecz]))
     equatorial_velocities = np.array(ecliptic_to_equatorial([dx, dy, dz]))
 
-    equatorial_coords += np.array((sun.x, sun.y, sun.z))
-    equatorial_velocities += np.array((sun.vx, sun.vy, sun.vz))
+    if orbit_format == "KEP" or orbit_format == "COM" or orbit_format == "CART":
```

Comment:
Totally optional, but it might be more tidy to make this
```suggestion
    if orbit_format in ["KEP", "COM", "CART"]:
```

Reply:
I completely agree!

Reply:
Updated this line.